eng/code-quality-reports/src/main/resources/checkstyle/checkstyle-suppressions.xml

```diff
@@ -89,7 +89,7 @@
   <suppress checks="FinalClass" files="Util.java"/>
 
   <!-- Star imports are ok in unit tests - it makes it easier to import all assertions, etc -->
-  <suppress checks="AvoidStarImport" files=".*[/\\]src[/\\]test[/\\]java[/\\].*\.java"/>
+<!--  <suppress checks="AvoidStarImport" files=".*[/\\]src[/\\]test[/\\]java[/\\].*\.java"/>-->
```

Comment:
Will revert these file changes once a green CI proving samples and test files cleaned up.